components/FilterList/index.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function FilterList({
+  muscleGroups,
+  onSelect,
+  selectedMuscleGroups,
+  onDeselect,
+  onClear,
+}) {
+  return (
+    <>
+      <MuscleList>
+        {muscleGroups.map((muscleGroup) => (
+          <MuscleTags key={muscleGroup} onClick={() => onSelect(muscleGroup)}>
+            {muscleGroup}
+          </MuscleTags>
+        ))}
+      </MuscleList>
```

Comment:
```suggestion
      </MuscleTagList>
```
Because your component should describe what it does